build.proj

```diff
@@ -25,11 +25,11 @@
   </Target>
 
   <PropertyGroup>
-    <OptDataProjectJson>$(SourceDir).nuget/optdata/project.json</OptDataProjectJson>
+    <OptDataCsproj>$(SourceDir).nuget/optdata/optdata.csproj</OptDataCsproj>
     <OptDataPackageFeed>https://dotnet.myget.org/F/dotnet-core-optimization-data/api/v3/index.json</OptDataPackageFeed>
   </PropertyGroup>
   <Target Name="RestoreOptData">
-    <Exec Condition="Exists('$(OptDataProjectJson)')" Command="$(DnuRestoreCommand) &quot;$(OptDataProjectJson)&quot; --source &quot;$(OptDataPackageFeed)&quot;" />
```

Comment:
Looks like the old project.json file doesn't even exist so I suspect this restore hasn't worked for a long time.